ui/pages/asset/components/asset-price.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, forwardRef, useImperativeHandle } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  Display,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { Box, Text } from '../../../components/component-library';
+import { formatCurrency } from '../../../helpers/utils/confirm-tx.util';
+import { getCurrentCurrency } from '../../../selectors';
+import { getPricePrecision, shortDateFormatter } from './util';
+
+const chartUp = (
+  <svg
+    width="12"
+    height="12"
+    viewBox="0 0 12 12"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M9.75 3.8125L6.25 7.4875L4.91667 5.3875L2.25 8.1875"
+      stroke="#28A745"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M8.08398 3.8125H9.75065V5.5625"
+      stroke="#28A745"
```

Comment:
@georgewrmarshall Do we have CSS vars we can use here instead of hard-coded hex values?

Reply:
Yes! `var(--color-success-default)`